README.md

```diff
@@ -75,3 +75,18 @@ Then, you can type:
 ```
 yarn gh-pages
 ```
+
+## Docker
+- Install docker
+- `cd` to `europa-component-library` project folder
+- Build: `docker build -v $PWD:/usr/src/app -t {username}/europa-component-library .`
```

Comment:
I resolve this issue in the last commit and I aligned the logic of this docker with the Docker logic.